medusa/clients/torrent/generic.py

```diff
@@ -44,67 +44,30 @@ def __init__(self, name, host=None, username=None, password=None):
-
-    def _request(self, method='get', params=None, data=None, files=None, cookies=None):
-
-        if time.time() > self.last_time + 1800 or not self.auth:
-            self.last_time = time.time()
-            self._get_auth()
-
-        text_params = str(params)
-        text_data = str(data)
-        text_files = str(files)
-        log.debug(
-            '{name}: Requested a {method} connection to {url} with'
-            ' params: {params} Data: {data} Files: {files}', {
-                'name': self.name,
-                'method': method.upper(),
-                'url': self.url,
-                'params': text_params[0:99] + '...' if len(text_params) > 102 else text_params,
-                'data': text_data[0:99] + '...' if len(text_data) > 102 else text_data,
-                'files': text_files[0:99] + '...' if len(text_files) > 102 else text_files,
-            }
-        )
-
-        if not self.auth:
-            log.warning('{name}: Authentication Failed', {'name': self.name})
+        self.verify = certifi.where()
+        
+        if self._get_auth():
             return False
 
-        try:
-            self.response = self.session.request(method, self.url, params=params, data=data, files=files,
-                                                 cookies=cookies, timeout=120, verify=False)
-        except (requests.exceptions.MissingSchema, requests.exceptions.InvalidURL) as error:
-            log.warning('{name}: Invalid Host: {error}', {'name': self.name, 'error': error})
-            return False
-        except requests.exceptions.RequestException as error:
-            log.warning('{name}: Error occurred during request: {error}',
-                        {'name': self.name, 'error': error})
-            return False
-        except Exception as error:
-            log.error('{name}: Unknown exception raised when sending torrent to'
-                      ' {name}: {error}', {'name': self.name, 'error': error})
-            return False
+    def _request(self, method='get', params=None, data=None, files=None, cookies=None):
 
+        self.response = self.session.request(method, self.url, params=params, data=data, files=files, timeout=60, verify=self.verify)
         if not self.response:
-            log.warning('{name}: Unable to reach torrent client', {'name': self.name})
-            return False
-
-        if self.response.status_code == 401:
-            log.error('{name}: Invalid Username or Password, check your config', {'name': self.name})
+            log.warning(f"{self.name} {method.upper()} call to {self.url} failed!")
+            self.message = f'Connect to {self.name} on "{self.host}" failed!'
             return False
-
-        code_description = http_code_description(self.response.status_code)
-        if code_description is not None:
-            log.info('{name}: {code}', {'name': self.name, 'code': code_description})
+        if self.response.status_code  >= 400:
```

Comment:
linter complains about double space

Reply:
Don't know why, this is perfectly fine python code.